diff --git a/node_modules/@expo/webpack-config/webpack/plugins/ExpoDefinePlugin.js b/node_modules/@expo/webpack-config/webpack/plugins/ExpoDefinePlugin.js
index 0145e72..34a36ff 100644
--- a/node_modules/@expo/webpack-config/webpack/plugins/ExpoDefinePlugin.js
+++ b/node_modules/@expo/webpack-config/webpack/plugins/ExpoDefinePlugin.js
@@ -57,8 +57,7 @@ function createClientEnvironment(mode, publicPath, nativeAppManifest, platform)
     const __DEV__ = environment !== 'production';
     // Adding the env variables to the Expo manifest is unsafe.
     // This feature is deprecated in SDK 41 forward.
-    const isEnvBindingSupported = lteSdkVersion(nativeAppManifest, '40.0.0');
-    const ENV_VAR_REGEX = isEnvBindingSupported ? /^(EXPO_|REACT_NATIVE_|CI$)/i : /^(CI$)/i;
+    const ENV_VAR_REGEX = /^(EXPO_|REACT_NATIVE_|CI$)/i;
     const SECRET_REGEX = /(PASSWORD|SECRET|TOKEN)/i;
     const shouldDefineKeys = (0, getenv_1.boolish)('EXPO_WEBPACK_DEFINE_ENVIRONMENT_AS_KEYS', false);
     const prefix = shouldDefineKeys ? 'process.env.' : '';
